internal/api/shared/migrate/migrate_test.go

```diff
@@ -16,41 +16,43 @@ package migrate
 import (
```

Comment:
I first started to add a new test in the migration part, with some refactor at the same time, before figuring out that it was a validation issue and not a migration issue. So in the end I moved the test in the `validate` package but still kept the refactoring here